uxarray/io/utils.py

```diff
@@ -37,6 +37,8 @@ def _parse_grid_type(dataset):
         mesh_type = "MPAS"
     elif "maxNodePElement" in dataset.dims:
         mesh_type = "ESMF"
+    elif all(key in dataset.sizes for key in ["nf", "YCdim", "XCdim"]):
```

Comment:
I think this check might not always work, as some meshes that have a uniform number of nodes per element, maxNodePElement might not be explicitly stated.

Also, maybe put a comment or ref for what these keys mean. 

```
nf: This is the number of faces in the grid. For a cubed-sphere grid, this is typically 6 (representing the 6 faces of a cube).

YCdim: This is the Y-coordinate dimension. It represents the number of grid cells in the Y direction on each face of the cubed-sphere.

XCdim: This is the X-coordinate dimension. It represents the number of grid cells in the X direction on each face of the cubed-sphere.
```

Reply:
Good suggestion!

Reply:
> I think this check might not always work, as some meshes that have a uniform number of nodes per element, maxNodePElement might not be explicitly stated.

The `maxNodePElement` is not part of this PR, but I agree there are some meshes that are stored in a 1D format as opposed to a 2D array.